commands/v1.proto

```diff
@@ -68,12 +68,23 @@ message KillResponse {
 
 // Command message sent to santa.host.<deviceid>.commands
 message SantaCommandRequest {
+  // Nonce
   string uuid = 1;
 
   oneof command {
     PingRequest ping = 10;
     KillRequest kill = 11;
   }
+
+  // timestamp in ms to prevent replays
+  int64 issued_at_ms = 2;
```

Comment:
Suggest we change to just a "second since epoch" interface, we don't really need millisecond precision.